config/environments/development.rb

```diff
@@ -68,7 +68,7 @@
   config.assets.quiet = true
 
   # Raises error for missing translations.
-  # config.i18n.raise_on_missing_translations = true
+  config.i18n.raise_on_missing_translations = true
```

Comment:
Oooo, danger zone! (But good, I think?)

Reply:
I didn't mean to commit this. (Although in development it is kinda helpful?)

I reverted this.